src/main/java/net/ltxprogrammer/changed/VariantCheck/PatienceCompatibility.java

```diff
@@ -0,0 +1,70 @@
+package net.ltxprogrammer.changed.VariantCheck;
```

Comment:
Package name doesn't fit the format neighboring packages use, specifically the capitalization.